VladShalamai/src/contactlist/ContactList.java

```diff
@@ -0,0 +1,169 @@
+package contactlist;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by Влад on 22.11.2016.
+ */
+public class ContactList {
+
+    public static final int MAX_SIZE = 20;
+    private String name;
+    private Contact[] contacts = new Contact[MAX_SIZE];
+
+    public ContactList(String name) {
+        this.name = name;
+    }
+
+    public Contact[] getContacts() {
+        return contacts;
+    }
+
+    public void setContacts(Contact[] contacts) {
+        this.contacts = contacts;
+    }
+
+    public String showAllContacts() {
+        StringBuilder sb = new StringBuilder();
+
+        for (int i = 0; i < MAX_SIZE; i++) {
+            if (contacts[i] != null) {
+                sb.append(contacts[i].showInfo()).append("\n");
+            }
+        }
+
+        return sb.toString();
+
+    }
+
+    public String showFirstFiveContacts() {
+
+        StringBuilder sb = new StringBuilder();
+
+        for (int i = 0; i < 5; i++) {
+            if (contacts[i] != null) {
```

Comment:
You should to keep some variable that will know about actual size of elements excluding null values
It simplify adding and showing by removing if(not null) condition